Formula/llvm.rb

```diff
@@ -158,6 +142,8 @@ def install
     end
 
     if OS.linux?
+      # The runtimes build can't find Glibc headers for some reason
+      ENV.append "CPPFLAGS", "-isystem /usr/include/x86_64-linux-gnu"
```

Comment:
Still getting missing headers on Linux even after adding this.

Reply:
>x86_64

Seems to be unportable. Also, on Linux there is an optional dependency on glibc [at line 43](https://github.com/Homebrew/homebrew-core/blob/a5c8d94db6a84362ab8471090eab58c13decf635/Formula/llvm.rb#L43) if it's installed:

```
depends_on "glibc" if Formula["glibc"].any_version_installed?
```